packages/open-next/src/overrides/incrementalCache/multi-tier-ddb-s3.ts

```diff
@@ -40,9 +41,8 @@ const awsFetch = (body: RequestInit["body"], type: "get" | "set" = "get") => {
   );
 };
 
-const buildDynamoKey = (key: string) => {
-  const { NEXT_BUILD_ID } = process.env;
-  return `__meta_${NEXT_BUILD_ID}_${key}`;
+const buildDynamoKey = (key: CacheKey<CacheEntryType>) => {
+  return `__meta_${key.buildId ? `${key.buildId}_` : ""}${key.baseKey}`;
```

Comment:
nit: could this be simplified?
not the exact same output as the current code but it should not matter?

```suggestion
  return `__meta_${key.buildId ?? ""}_${key.baseKey}`;
```